### PR TITLE
Add Server and Patient Access obligations

### PR DESCRIPTION
**Pull request by dbojicic**
Add Server and Patient Access obligations as agreed in https://confluence.hl7.org/pages/viewpage.action?pageId=256181872. 

Noting another PR will be submitted with agreed interactions changes.

### CODE REVIEW

File: input/fsh/au-erequesting-servicerequest-imag.fsh [hunk 1]
```diff
@@ -18,10 +18,21 @@ Description: "This profile sets minimum expectations for a ServiceRequest resour
 * category contains imaging 1..1
 * category[imaging] = $sct#363679005
 
+* code from https://ranzcr.com/fhir/ValueSet/radiology-referral (preferred)
```
Comment (dt-r) [inline input/fsh/au-erequesting-servicerequest-imag.fsh:21]: RANZCR PR content should not be in this PR. Remove.

File: input/fsh/au-erequesting-servicerequest-imag.fsh [hunk 2]
```diff
@@ -18,10 +18,21 @@ Description: "This profile sets minimum expectations for a ServiceRequest resour
...
 * bodySite MS
-
-
+* bodySite ^extension[http://hl7.org/fhir/StructureDefinition/obligation][0].extension[code].valueCode = #SHALL:populate-if-known
+* bodySite ^extension[http://hl7.org/fhir/StructureDefinition/obligation][0].extension[actor][0].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-placer"
+* bodySite ^extension[http://hl7.org/fhir/StructureDefinition/obligation][1].extension[code].valueCode = #SHALL:handle
+* bodySite ^extension[http://hl7.org/fhir/StructureDefinition/obligation][1].extension[actor].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-filler"
+* bodySite ^extension[http://hl7.org/fhir/StructureDefinition/obligation][2].extension[code].valueCode = #SHALL:handle
+* bodySite ^extension[http://hl7.org/fhir/StructureDefinition/obligation][2].extension[actor].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-server"
+* bodySite ^extension[http://hl7.org/fhir/StructureDefinition/obligation][2].extension[documentation].valueMarkdown = "For create/update ServiceRequest."
```
Comment (dt-r) [inline input/fsh/au-erequesting-servicerequest-imag.fsh:33]: Remove .valueMarkdown.

File: input/fsh/au-erequesting-servicerequest-imag.fsh (hunk 1 above)
Comment (dbojicic) [reply input/fsh/au-erequesting-servicerequest-imag.fsh:21]: Ah! Thanks for spotting this, will do now.

File: input/fsh/au-erequesting-servicerequest-path.fsh [hunk 3]
```diff
@@ -23,5 +23,15 @@ Description: "This profile sets minimum expectations for a ServiceRequest resour
...
 * bodySite MS
-
-
+* bodySite ^extension[http://hl7.org/fhir/StructureDefinition/obligation][0].extension[code].valueCode = #SHALL:populate-if-known
+* bodySite ^extension[http://hl7.org/fhir/StructureDefinition/obligation][0].extension[actor][0].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-placer"
+* bodySite ^extension[http://hl7.org/fhir/StructureDefinition/obligation][1].extension[code].valueCode = #SHALL:handle
+* bodySite ^extension[http://hl7.org/fhir/StructureDefinition/obligation][1].extension[actor].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-filler"
+* bodySite ^extension[http://hl7.org/fhir/StructureDefinition/obligation][2].extension[code].valueCode = #SHALL:handle
+* bodySite ^extension[http://hl7.org/fhir/StructureDefinition/obligation][2].extension[actor].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-server"
+* bodySite ^extension[http://hl7.org/fhir/StructureDefinition/obligation][2].extension[documentation].valueMarkdown = "For create/update ServiceRequest."
```
Comment (dt-r) [inline input/fsh/au-erequesting-servicerequest-path.fsh:32]: Remove the .valueMarkdown instances

File: input/fsh/au-erequesting-servicerequest.fsh [hunk 4]
```diff
@@ -13,47 +13,103 @@ Description: "This profile sets minimum expectations for a ServiceRequest resour
...
 * identifier ^extension[http://hl7.org/fhir/StructureDefinition/obligation][1].extension[actor].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-filler"
+* identifier ^extension[http://hl7.org/fhir/StructureDefinition/obligation][2].extension[code].valueCode = #SHALL:handle
+* identifier ^extension[http://hl7.org/fhir/StructureDefinition/obligation][2].extension[actor].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-server"
+* identifier ^extension[http://hl7.org/fhir/StructureDefinition/obligation][2].extension[documentation].valueMarkdown = "For create/update ServiceRequest."
+* identifier ^extension[http://hl7.org/fhir/StructureDefinition/obligation][3].extension[code].valueCode = #SHALL:able-to-populate
+* identifier ^extension[http://hl7.org/fhir/StructureDefinition/obligation][3].extension[actor].valueCanonical = "http://hl7.org.au/fhir/ereq/ActorDefinition/au-erequesting-actor-server"
+* identifier ^extension[http://hl7.org/fhir/StructureDefinition/obligation][3].extension[documentation].valueMarkdown = "For search/read ServiceRequest."
```
Comment (dt-r) [inline input/fsh/au-erequesting-servicerequest.fsh:21]: Remove .valueMarkdown instances

File: input/fsh/au-erequesting-servicerequest-path.fsh (hunk 3 above)
Comment (dbojicic) [reply input/fsh/au-erequesting-servicerequest-path.fsh:32]: Done

File: input/fsh/au-erequesting-servicerequest-imag.fsh (hunk 2 above)
Comment (dbojicic) [reply input/fsh/au-erequesting-servicerequest-imag.fsh]: Done

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments

File: input/fsh/au-erequesting-servicerequest-path.fsh (hunk 3 above)
Comment (dt-r) [reply input/fsh/au-erequesting-servicerequest-path.fsh:32]: Missed one in this file on bodySite

**Review by dt-r — CHANGES_REQUESTED** [review]
Missed one removal - see unresolved comment.

File: input/fsh/au-erequesting-servicerequest-path.fsh (hunk 3 above)
Comment (dbojicic) [reply input/fsh/au-erequesting-servicerequest-path.fsh]: Removed now.

**Review by dt-r — APPROVED** [review]

**Pull request merged**